content/en/service_management/workflows/track.md

```diff
@@ -0,0 +1,118 @@
+| `datadog.workflows.executions.completed`       | Number of workflow instances that have completed.      |
+| `datadog.workflows.steps.executions.started`   | Number of workflow instance steps that have started.   |
+| `datadog.workflows.steps.executions.completed` | Number of workflow instance steps that have completed. |
+
+To create a monitor that tracks whether daily workflow executions exceed a certain threshold, perform the following steps:
+
+1. Go to [New Monitor][4] and select the **Metric** monitor type.
+1. Under **Define the metric**, for **a**, fill in `datadog.workflows.executions.started`.
+1. _Optionally_, to restrict the monitor to a specific workflow, for **from**, fill in `workflow_id:[WORKFLOW-ID]`, replacing `[WORKFLOW-ID]` with the ID for your workflow.
+1. For **Evaluation Details**, use the following values:
+    - **Evaluate the**: `sum`
+    - **Of the query over the**: `last 1 day`.
+1. For **Set alert conditions**, choose **above**, then fill in an alert and warning threshold. For example, you could fill in an **Alert threshold** of `200` and a **Warning threshold** of `150`.
+1. Under **Configure notifications & automations**, name your workflow, then fill in message text. For example, you could use message text like the following:
+
+    {{< code-block lang="none" >}}@slack-alert-channel
+
+{{#is_warning}}
+Workflow has executed {{warn_threshold}} times in the last day; manual action might be needed to avoid alerting.
+{{/is_warning}}
+{{#is_alert}}
+Workflow has executed {{threshold}} times in the last day, which is our budget threshold for workflows. We should unpublish the workflow to avoid any more automatic executions for the day.
+{{/is_alert}}
+{{< /code-block >}}
+1. Click **Create**.
+
+
+## View workflow events in Event Manager
+
+You can use [Event Manager][5] to view workflow start and completion events by filtering on `source:workflow_automation`.
+
+To see events for a specific workflow, in the **Search facets** box, search for `workflow.workflow_id`. You can select a specific set of IDs to view only events for those workflows.
+
+You can also filter the output by **Status** to see only `info`, `warn`, or `error` messages.
+
+
+## Track workflow billing in Usage Attribution
+
+<div class="alert alert-warning">
+Usage Attribution is an advanced feature included in the Enterprise plan. For all other plans, contact your account representative or <a href="mailto:success@datadoghq.com">success@datadoghq.com</a> to request this feature.
+</div>
+
+To track billing for your workflow executions, perform the following steps:
+
+1. Navigate to the [Usage Attribution][6] page.
+1. Under **Products** on the left, search for **Workflow Executions**.
+1. Hover over **Workflow Executions** and click **Only** to view usage attribution for only workflows.<br>A list of workflow execution counts organized by organization, data center, and team appears.
```

Comment:
the organization, datacenter, and team are custom tags. I think usage attribution can be filtered by any tag? can you doublecheck their docs? You are only seeing those 3 because that's what the demo org has set up i think

Reply:
Ah, good to know! I'll take out specifics there.